### ISSUE TITLE
Update the eclipselink properties in persistence.xml to jakarta properties

### ISSUE THREAD

**Issue by gkwan-ibm** [OpenLiberty/guide-liberty-deep-dive#138]
Determine the min set of properties 

change
```
        <property name="eclipselink.target-database" value="PostgreSQL"/>
        <property name="eclipselink.ddl-generation" value="create-or-extend-tables" /> 
        <property name="eclipselink.logging.level" value="FINE"/>
        <property name="eclipselink.jdbc.url" value="jdbc/postgresql:/admin"/>         
        <property name="eclipselink.persistence.jdbc.driver" value="org.postgresql.Driver"/>  
        <property name="eclipselink.persistence.jdbc.user" value="admin"/>  
        <property name="eclipselink.persistence.jdbc.password" value="adminpwd"/> 
```
to
```
        <property name="jakarta.persistence.schema-generation.database.action"
                  value="create"/>
        <property name="jakarta.persistence.schema-generation.scripts.action"
                  value="create"/>
        <property name="jakarta.persistence.schema-generation.scripts.create-target"
                  value="createDDL.ddl"/>
```

**Comment by shin19991207** [comment]
addressed in #139, no issue

**Comment by gkwan-ibm** [comment]
close
- issue was addressed
